batchflow/tests/named_expr_test.py

```diff
@@ -94,16 +99,16 @@ def test_apply_parallel_p(named_expr, src):
     else:
         assert (b.images == b.masks).all()
```

Comment:
can't understand how this test passes, because b.images are supposed to be randomly changed in `ap_test`...

Reply:
The very same transformation is applied to both `images` and `masks`

Reply:
And the case where only `images` are transformed is checked in other branch